### PR TITLE
[PREVIEW] Feature/sl 1114/view additional fields listing request

### PR DESCRIPTION
**Pull request by r-kruk-kainos**

### CODE REVIEW

**Comment by satyachundur** [comment]
@r-kruk-kainos -  could you look at linting issue failing the build.  Once its building and creating preview will verify this PR again

File: src/app/notes/components/notes-list-dialog/notes-list-dialog.component.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div id="notesDialog" draggable="true" class="container">
+    <app-note-list [notes]="data" [disabled]=true></app-note-list>
+    <div style="margin-top:30px;">
```
Comment (zielinskikamil) [inline src/app/notes/components/notes-list-dialog/notes-list-dialog.component.html]: you can move it to .scss file :)

File: src/app/hearing-part/components/hearing-parts-preview/hearing-parts-preview.component.ts [hunk 2]
```diff
@@ -53,6 +58,22 @@ export class HearingPartsPreviewComponent implements OnInit, OnChanges {
...
+    }
+
+    openNotesDialog(hearingPart: HearingPartViewModel) {
+        if (this.hasNotes(hearingPart)) {
+            this.dialog.open(NotesListDialogComponent, {
+                data: hearingPart.notes,
+                hasBackdrop: false,
+                width: '30%'
+            })
+        } else {
```
Comment (leszekgu) [inline src/app/hearing-part/components/hearing-parts-preview/hearing-parts-preview.component.ts]: why this else part? is it required by sonarQube or some check-style?

File: src/app/hearing-part/models/hearing-part.viewmodel.ts [hunk 3]
```diff
@@ -1,14 +1,22 @@
...
-import { Session } from '../../sessions/models/session.model';
+import { Note } from '../../notes/models/note.model';
+import { Priority } from './priority-model';
+import { Judge } from '../../judges/models/judge.model';
 
-export interface HearingPartViewmodel {
+export interface HearingPartViewModel {
     id: string;
-    session: Session;
+    session: string;
```
Comment (leszekgu) [inline src/app/hearing-part/models/hearing-part.viewmodel.ts]: later on, I would suggest changing the field name, if it's not a full session anymore (is it id?)

**Pull request merged**